src/ipvs/ip_vs_conhash.c

```diff
@@ -182,17 +180,13 @@ static int dp_vs_conhash_add_dest(struct dp_vs_service *svc,
     }
 
     INIT_LIST_HEAD(&(p_conhash_node->list));
-    p_conhash_node->af = dest->af;
-    p_conhash_node->addr = dest->addr;
-    p_conhash_node->port = dest->port;
-    p_conhash_node->weight = weight;
 
     // add node to conhash
     p_node = &(p_conhash_node->node);
-    addr_fold = inet_addr_fold(dest->af, &dest->addr);
-    snprintf(str, sizeof(str), "%u%d", addr_fold, dest->port);
+    inet_ntop(dest->af, &dest->addr, addr, sizeof(addr));
```

Comment:
for nat66, inet_addr_fold may cause two rs to have the same addr_fold.In this case, there will be an rs that can never be added to the rbtree;